src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/NavigationBar.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from 'react';
+import { Grid, makeStyles, createMuiTheme } from '@material-ui/core';
+import { useDispatch, useSelector } from 'react-redux';
+import { IRuntimeState, Triggers } from 'src/types';
+import { FormLayoutActions } from 'src/features/form/layout/formLayoutSlice';
+import { getTextResource } from '../../utils/formComponentUtils';
+import { AltinnAppTheme } from 'altinn-shared/theme';
+import { useState, useEffect } from 'react';
+
+const theme = createMuiTheme(AltinnAppTheme);
+
+const useStyles = makeStyles({
+  ul: {
```

Comment:
ClassNames should not have HTML tags as names.
This makes the code more complex to maintain.